scout/data-handling.md

```diff
@@ -0,0 +1,57 @@
+---
+description: How Docker Scout handles image metadata
+keywords: scanning, supply chain, security, data, metadata
+title: Data collection and storage in Docker Scout
+---
+
+{% include scout-early-access.md %}
+
+Docker Scout image analysis works by collecting metadata from the container
+images that you analyze. That metadata is stored on the Docker Scout platform.
```

Comment:
```suggestion
images that you analyze. This metadata is stored on the Docker Scout platform.
```
This? That? Thems? Thine? Thee? I think 'This' reads better, but I'm being picky